Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1343,9 +1363,30 @@
   icon: { sprite: /Textures/Objects/Weapons/Melee/e_shield.rsi, state: eshield-on }
   productEntity: EnergyShield
   cost:
-    Telecrystal: 6
+    Telecrystal: 8
   categories:
   - UplinkMisc
+  conditions:
+  - !type:StoreWhitelistCondition
+    blacklist:
+      tags:
+      - NukeOpsUplink
+
+- type: listing
+  id: UplinkEshieldNukie
+  name: uplink-eshield-name
+  description: uplink-eshield-desc
+  icon: { sprite: /Textures/Objects/Weapons/Melee/e_shield.rsi, state: eshield-on }
+  productEntity: EnergyShield
+  cost:
+    Telecrystal: 12
+  categories:
+  - UplinkMisc
+  conditions:
+  - !type:StoreWhitelistCondition
+    whitelist:
+      tags:
+      - NukeOpsUplink
```

Comment:
Traitors shouldn't have access to the regular EShield and it should just be upped to 8 TC in general.

Reply:
There does need to be a price nerf, but its an item that has 200 HP, and high reflection chance, I'm not sure 8 TC is high enough.

Reply:
It's meant to have high HP since you're spending money and can't just replace or reprint it. Higher than 8 starts to get into territory of "is it really worth buying this consumable vs the HP cost". Maybe we try 8tc first, and if it's still too little we do another increment to 10. Then ultimately 12 if it's still too little.

Reply:
Yeah it can always be adjusted. Sure.